config/configurator.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import "github.com/spf13/viper"
+
+type configurationLoader interface {
+	setConfig(name string, extension string, path string)
+	read() error
+	merge() error
+}
+
+type configurationReader interface {
```

Comment:
As it's inside `config` package, I think just `Reader` and `Loader` would be enough.

Reply:
Done.